docs/product_management_and_merchandising/accommodation_and_experience_products.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Accommodation and Experience Products
+parent: Product Management and Merchandising
+nav_order: 3
+---
+
+Accommodation type products have one key difference which is a Daily Availability & Pricing feature. This feature expects what is being sold to be available across many dates, and that the customer should be able to choose check in and check out dates, perhaps with a minimum nights' stay.
```

Comment:
Perhaps there should be a quick line just saying Products can be either 'experiences' or 'accommodations'?